feinstaub/nginx.conf.sigil

```diff
@@ -11,9 +11,10 @@ server {
   {{ if $.NOSSL_SERVER_NAME }}server_name {{ $.NOSSL_SERVER_NAME }}; {{ end }}
   access_log  /var/log/nginx/{{ $.APP }}-access.log;
   error_log   /var/log/nginx/{{ $.APP }}-error.log;
-{{ if (and (eq $listen_port "80") ($.SSL_INUSE)) }}
-  return 301 https://$host:{{ $.NGINX_SSL_PORT }}$request_uri;
-{{ else }}
+
+  location /static/ {
+    alias /home/feinstaub-data/feinstaub/static/;
+  }
```

Comment:
Wouldn't this be defined in Django instead?

Reply:
We certainly do not need to specify this here again.